ccuapi/__init__.py

```diff
@@ -23,6 +23,7 @@
 AKAMAI_NOTIFY_EMAIL = os.environ.get('AKAMAI_NOTIFY_EMAIL', None)
 AKAMAI_API_HOST = os.environ.get('AKAMAI_API_HOST',
                                  'api.ccu.akamai.com')
+AKAMAI_HTTPS_TIMEOUT = 34
```

Comment:
Is there anything special about the value of 34? I think we should make this load from the ENV, and default to 34, similar to `AKAMAI_API_HOST`